src/rewriter/README.md

```diff
@@ -85,7 +103,7 @@ All used variables in match and target must show up in the parameters list and
 	(concat (repeat n (extract (- s 1) (- s 1) x)) x))
 ```
 
-## CVC5 Implementation Details
+## Adding new operators
```

Comment:
I like this section. Maybe add to the heading (or text) that this is purely about parsing RARE rules in cvc5? Since the rest of the documentation is not cvc5 specific.

Reply:
done